### PR TITLE
Thellimist/pi 1060 add drawer to all cards

### PR DESCRIPTION
**Pull request by thellimist**
Some texts don't use markdown. I'll double check that with Garreth

### CODE REVIEW

File: src/components/FormattedMarkdown/FormattedMarkdown.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    //
+    // 1) formatMessage({ id: mycomponent.myfeature.integrationName })
+    // 2) formatMessage({ id: mycomponent.myfeature })
+    // 3) "mycomponent.myfeature"
+    //
+    // NOTE: This will log errors in dev but not in production
+    var formattedMessage = formatMessage({
+      id: this.props.text + '.' + integrationName,
+      defaultMessage: formatMessage({ id: this.props.text })
+    });
```
Comment (thellimist) [inline src/components/FormattedMarkdown/FormattedMarkdown.js]: This approach is annoying in dev cause of console.errors's printed. But it is a viable solution to support translations for different integrations. We can use this approach for `card.title.wordpress`, `card.title.magento` and similar translations

Comment (jwineman) [inline src/components/FormattedMarkdown/FormattedMarkdown.js]: So you're basically assuming one of these will fail? What about [this](https://github.com/yahoo/react-intl/issues/747)?
```
let integrationKey = this.props.text + '.' + integrationName;
const stringExists = !!this.props.intl.messages[integrationKey];
var formattedMessage = formatMessage({id: stringExists ? integrationKey : this.props.text});
```

**Comment by thellimist** [comment]
We needed different text for different plugins. In 6a8c08f and 9fdfe6a I changed the text of Apply Default Settings. I thought of using `%{integrationName}` as parameter but this seemed like a better approach. It gives more versatility.

File: package.json [hunk 2]
```diff
@@ -70,6 +70,7 @@
     "intl": "^1.1.0",
     "jest": "^19.0.2",
     "lodash": "^3.10.1",
+    "marked": "^0.3.6",
```
Comment (sejoker) [inline package.json]: the package is abandoned. I would recommend using [alternative](https://www.npmjs.com/package/markdown-it)

**Pull request merged**